### DISCUSSION TITLE
Funzione scrutinio DAD

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#143] [Richieste di aiuto]
Buongiorno,ho notato che molti registri , a causa del covid , durante lo scrutinio che ormai si sta svolgendo sempre a distanza , i docenti , siccome non possono firmare il tabellone direttamente in modalità cartaceo , ogni docente nel registro ,nella sezione tabellone dello scrutinio risulta avere  una matita ,cliccando la quale ,inserisce la stessa password che utilizza per accedere e firma in modo digitale e veloce il tabellone .Si potrebbe pensare ad una cosa simile anche per Giuaschool?

**Comment by trinko** [comment, marked as answer]
Al momento no. Non è una modifica semplice.
